packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -383,12 +376,10 @@ abstract class BaseFlutterTask extends DefaultTask {
         if (buildMode != 'debug') {
             // For AOT builds, include the gen_snapshot depfile.
             FileCollection depfiles = project.files("${intermediateDir}/snapshot.d")
-            if (previewDart2) {
-                // For Dart 2, also include the kernel compiler depfile, since
-                // kernel compile is the first stage of AOT build in this mode,
-                // and it includes all the Dart sources.
-                depfiles += project.files("${intermediateDir}/kernel_compile.d")
-            }
+            // For Dart 2, also include the kernel compiler depfile, since
```

Comment:
You can just say "Include the kernel...", since there is no Dart 1...

Reply:
Done!